frontend/src/app/shared/components/file-links/storage-information/storage-information.html

```diff
@@ -18,10 +18,19 @@
 <div
   class="button-box"
 >
-  <button class="button button_no-margin">
-    <op-icon icon-classes="button--icon spot-icon spot-icon_external-link" class="op-icon--wrapper">
-      <i class="button--icon spot-icon spot-icon_external-link"></i>
+  <button *ngFor="let button of buttons"
```

Comment:
Since I could not look at it in the browser, this is just guesswork, but I assume that two buttons next to each other won't have any space in between them.

Reply:
Sadly there is no easy way in our Angular application to denote that there is just one button or nothing at all. I tried using a Promise for that case, but as I need to access multiple properties of the `StorageActionButton`, I would had to introduce the subscription login in the component itself instead of using the `async` pipe for this.

But I agree, that if there were multiple buttons, they would look not correct. But we have no style for this. So i left it out until it was necessary. If someone puts two buttons there, then there will be a style how to do this.

Reply:
You don't need Angular for that. You can let the `button-box` handle that case via CSS. Either you display the box as a `grid` with `grid-column-gap: XYpx` or you add a `margin-right` to all buttons except for the last one (`:last-of-type`). https://github.com/opf/openproject/pull/10859#pullrequestreview-1021628584 already introduces a class for the buttons and add some margins any way. So you could simply extend that (once it is merged)